ddtrace/internal/datadog/profiling/cmake/FindCxxMsan.cmake

```diff
@@ -0,0 +1,22 @@
+include(ExternalProject)
+
+if(NOT TARGET llvm-libcxx-msan)
+    ExternalProject_Add(llvm-libcxx-msan
+        PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/llvm-libcxx-msan
+        GIT_REPOSITORY    https://github.com/llvm/llvm-project.git
+        GIT_TAG           llvmorg-17.0.6 # should probably reflect toolchain version!!!
```

Comment:
Does this mean we need to use 17.0.6 in ci?

Reply:
No--ideally the harness will detect the version of clang used here and get the right tag.  I still need to add that.